src/components/Causes/css.js

```diff
@@ -0,0 +1,79 @@
+import styled from 'styled-components';
+import EventPhoto from '../../images/womenintech-6.jpg';
+
+const CausesSection = styled.section`
+	@media (min-width: 850px) {
+		background-color: #ffffff;
+		position: relative;
+		padding: 5rem 1.25rem 8rem 1.25rem;
+	}
+
+	@media (max-width: 849px) {
+		background-image: url(${EventPhoto});
+		background-position: center center;
+		background-size: cover;
+	}
+`;
+
+const CausesImg = styled.section`
+	@media (min-width: 850px) {
+		background-image: url(${EventPhoto});
+		background-size: cover;
+		height: 500px;
+		width: 800px;
+		position: absolute;
+	}
+`;
+
+const CausesTextSection = styled.section`
+	@media (min-width: 1025px) {
+		background-color: #ffffff;
+		position: relative;
+		height: 450px;
+		width: 700px;
+		margin-left: 32rem;
+		padding: 3rem;
+		transform: translateY(1.5rem);
+	}
+
+	@media (min-width: 850px) and (max-width: 1024px) {
+		background-color: #ffffff;
+		position: relative;
+		height: 450px;
+		width: 700px;
+		margin-left: 13rem;
+		padding: 3rem;
+		transform: translateY(1.5rem);
+	}
+
+	@media (max-width: 849px) {
+		background: rgba(255, 255, 255, 0.75);
+		padding: 3rem;
+		text-align: center;
+	}
+`;
+
+const CausesH2 = styled.header`
```

Comment:
Instead of styled.header it's more semantically correct to make this styled.h2 :) 

headers are for the header section at the top. 

H2 makes more sense for a section heading :)

Reply:
OMG DUH - how did I miss this!?!?! LOOOOOL